src/app/model/Pawn.ts

```diff
@@ -0,0 +1,70 @@
+import { Board } from './Board';
+import { BOARDSIZE } from './Constances';
+import { Piece } from './Piece';
+import { PieceMovement } from './PieceMovement';
+import { Column, columns, Row, Side, Square } from './Types';
+
+export class Pawn extends Piece implements PieceMovement {
+  private nextRowDifference: number = 1;
+
+  constructor(id: string, side: Side) {
```

Comment:
Kto wywala w końcu to id :) ? Możesz je np. pozbyć się już w konstruktorze Pawn robiąc tak:
```
constructor(id: string, side: Side) {
super('uselessId', side)
```
A potem ktos to wywali na dobre z Piece :)

Reply:
zmienione